fhir-config/src/main/java/com/ibm/fhir/config/FHIRRequestContext.java

```diff
@@ -33,6 +33,8 @@
     private String tenantId;
     private String tenantKey;
     private String dataStoreId;
+    // By default, setting to False.
+    private Boolean dataStoreMultiTenant = Boolean.FALSE;
```

Comment:
trying to decide if we want this here...I guess is the idea is its similar to the tenantKey so put it in the same spot?

that one made sense to me because it was already here, but I'm not sure if we want to keep adding stuff to the requestcontext.  can we keep it isolated to just the FHIRDbDAOImpl?

Reply:
I think it makes sense here. it maintains state, and keeps the DAO somewhat stateless

Reply:
I guess I was thinking we could look up the config property where its needed without storing it as state anywhere.  It didn't seem to be used from anywhere other than FHIRDbDAOImpl.

Reply:
Right, but you end up looking up the property group again, and processing again.  Wouldn't it be good to pick it off one time?

Reply:
We do have caching for the config and I think its consistent with other places that read the config whenever we need a config value.
That said, if we're calling it over and over then maybe you're right...do we call getConnection() a lot of times during a typical create/update request?

Reply:
While that is true, the config is still navigated down the tree to the property, versus a point lookup. 

In general it's called in multiple spots, but if it's cached in memory (resourceType, parameterName et cetra it shouldn't hit that point), but it is referenced in a lot of places with many different paths.

Reply:
OK, I think its probably fine either way.  Would like to hear from @JohnTimm and @albertwang-ibm in terms of preference, but I think I'm good with whatever the group decides.

Reply:
I don’t have a strong opinion either way but if `tenantId` and `dsId` are already in the `FHIRRequestContext` then adding a derived field for convenience doesn’t bother me.

Reply:
I would agree with Paul's current codes.

Reply:
OK, marking it resolved then.  Thanks for weighing in.